### PR TITLE
Bump version to 4.0.0

### PR DESCRIPTION
**Pull request by benthorner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.0.0
```
Comment (theseanything) [inline CHANGELOG.md:1]: Sorry to be particular, but not sure this should be a major bump because we aren't changing the API of rubocop-govuk

Comment (theseanything) [reply CHANGELOG.md:1]: As in a people don't need to change how rubocop-govuk is configured in their repo.

Comment (theseanything) [reply CHANGELOG.md:1]: It does change the behaviour of repo - adding functionality of alerting about more linting errors. But isn't a breaking change in how you use rubocop-govuk

**Comment by benthorner** [comment]
@theseanything that reasoning also makes sense. This is fun: https://github.com/alphagov/rubocop-govuk/pull/53

**Pull request closed without merging**